server/game/TargetInterfaces.ts

```diff
@@ -82,7 +81,7 @@ export interface ITargetResolverBase<TContext extends AbilityContext> {
 // TODO: add functionality to PlayerTargetResolver to autodetect any invalid target players.
 export interface IPlayerTargetResolver<TContext extends AbilityContext> extends ITargetResolverBase<TContext> {
     mode: TargetMode.Player | TargetMode.MultiplePlayers;
-    immediateEffect?: PlayerTargetSystem<TContext>;
+    immediateEffect?: GameSystem<TContext>;
```

Comment:
Had to change this to be able to use a conditional effect with a player target resolver.